### PR TITLE
Make support (discourse and slack) more prominent

### PR DESCRIPTION
**Pull request by mschauer**

### CODE REVIEW

**Comment by mschauer** [comment]
I just see there is some overlap with #899

File: README.md [hunk 1]
```diff
@@ -23,16 +23,18 @@ A Julia package for probability distributions and associated functions. Particul
...
 ## Contributing
 
-### Reporting issues & discussing
+### Reporting issues
 
-1. If you need explanation on how to do X, Y using Distributions,
-feel free to ask on the Julia [Discourse](https://discourse.julialang.org/c/domain/stats)
-or [Slack](https://julialang.slack.com), get an invitation
-[here](https://slackinvite.julialang.org/).
+* If you need help or an explanation how to use *Distributions* ask in the forum (https://discourse.julialang.org) or, for informal questions, visit the chat (https://julialang.slack.com).
```
Comment (Peiffap) [inline README.md]: Why not format the links using the `[]()` syntax?

**Comment by matbesancon** [comment]
@mschauer seems ok since the other PR does not modify the readme

File: README.md (hunk 1 above)
Comment (mschauer) [reply README.md]: They are displayed as links this way as well.

**Pull request merged**